### DISCUSSION TITLE
Thank you for the new supported version!

### DISCUSSION THREAD

**Discussion by Mexflubber** [PasaOpasen/geneticalgorithm2-6.8.7#11] [Q&A]
Hi,

Thank you for the new supported version. Have you tested it in Mac Catalina ? I have a local branch that I had to modify (from geneticalgorithm) to make it work. 

I also added memoization. Can share the code here and let you merge it in, I would hate to do a bad job rebasing.

**Comment by PasaOpasen** [comment]
@Mexflubber sorry for waiting

I never used Mac, so I didn't test it on Mac, but I tested on my Windows and on Linux servers on Kaggle

If u wanna share some work, just let me know. What exactly did u with memoization?
